lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -219,6 +222,41 @@
                 },
 
                 /**
+                 * Render the edit team form.
+                 */
+                editTeam: function (topicID, teamID) {
+                    var self = this;
+                    this.getTeamsView(topicID).done(function (teamsView) {
+                        self.getTeam(teamID, false).done(function(team) {
+                            self.mainView = new ViewWithHeader({
+                                header: new HeaderView({
+                                    model: new TeamsHeaderModel({
+                                        description: gettext("You can edit the details of a team here, and should separately let your team members know about changes."),
+                                        title: gettext("Edit Team"),
+                                        breadcrumbs: [
+                                            {
+                                                title: teamsView.main.teamParams.topicName,
+                                                url: '#topics/' + teamsView.main.teamParams.topicID
+                                            }
+                                        ]
+                                    })
+                                }),
+                                main: new TeamEditView({
+                                    action: 'edit',
+                                    teamEvents: self.teamEvents,
+                                    tagName: 'create-new-team',
```

Comment:
If this is going to be the same in both cases, we don't need to make it an option.